### PR TITLE
Additional Fuel Tracking & Plotting

### PR DESCRIPTION
**Pull request by Sof222**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

**Comment by Sof222** [comment]
Happy New Year!
I have some more commits I would like to merge. 
Should I add to this pull request, or wait for it to be accepted and then open a new pull request?

**Comment by planes** [comment]
What sort of commits are you looking to add? If it fit's into the context of fuel tracking and plotting you may include it here. If it is different content another PR would be preferred. In general, we prefer smaller PR's as they're quick and easy.

**Comment by Sof222** [comment]
The new commits are for an additional component, so I will do a new PR after this one :)

File: regression/scripts/fuel_tracking_mission/fuel_tracking_mission.py [hunk 1]
```diff
@@ -0,0 +1,1041 @@
+# fuel_tracking_mission.py
+# 
+# Created:  Aug 2014, SUAVE Team
```
Comment (planes) [inline regression/scripts/fuel_tracking_mission/fuel_tracking_mission.py:3]: Update this to give yourself credit as the original author

File: regression/scripts/fuel_tracking_mission/fuel_tracking_mission.py [hunk 2]
```diff
@@ -0,0 +1,1041 @@
...
+#   Imports
+# ----------------------------------------------------------------------
+
+# General Python Imports
+import numpy as np
+
+import matplotlib.pyplot as plt
+
+# SUAVE Imports
+import SUAVE
```
Comment (planes) [inline regression/scripts/fuel_tracking_mission/fuel_tracking_mission.py:17]: remove the extra spaces between the imports

File: regression/scripts/fuel_tracking_mission/fuel_tracking_mission.py [hunk 3]
```diff
@@ -0,0 +1,1041 @@
...
+    atmosphere.features.planet = planet.features
+    analyses.append(atmosphere)   
+
+    return analyses    
+
+# ----------------------------------------------------------------------
+#   Define the Vehicle
+# ----------------------------------------------------------------------
+
+def vehicle_setup():
```
Comment (planes) [inline regression/scripts/fuel_tracking_mission/fuel_tracking_mission.py:162]: all vehicles should be in: https://github.com/suavecode/SUAVE/tree/develop/regression/scripts/Vehicles

File: regression/scripts/fuel_tracking_mission/fuel_tracking_mission.py [hunk 4]
```diff
@@ -0,0 +1,1041 @@
...
+    return missions  
+
+def check_results(results):
+    #checking the weight values at the final segement 
+
+    weights = results.segments[-1].conditions.weights
+
+
+    #stored weight values to check against results
+    total_mass                   = np.array([[62696.17099197],[62694.1442812 ],[62688.14554268],[62678.41600768],[62665.3481715],[62649.47228568],[62631.43866482],[62611.9962415 ],[62591.96729459],[62572.21788392],[62553.62354735],[62537.03040895],[62523.21294419],[62512.83097265],[62506.38959385],[62504.20634677]])
```
Comment (planes) [inline regression/scripts/fuel_tracking_mission/fuel_tracking_mission.py:1012]: You should just include this entire regression to an existing regression script.

File: trunk/SUAVE/Methods/Propulsion/serial_HTS_turboelectric_sizing.py [hunk 5]
```diff
@@ -1,10 +1,19 @@
-## @ingroup Methods-Propulsion
+# @ingroup Methods-Propulsion
 # serial_HTS_turboelectric_sizing.py
 # 
-# Created:  Mar 2020,   K. Hamilton
-# Modified: Nov 2021,   S. Claridge
+# Created:  K. Hamilton Mar 2020
```
Comment (planes) [inline trunk/SUAVE/Methods/Propulsion/serial_HTS_turboelectric_sizing.py:4]: It was right the first time

**Review by planes — COMMENTED** [review]
Good work!

See my comments

**Comment by Sof222** [comment]
Hi! 
Just checking if there were any other changes I needed to make? :)

**Review by planes — APPROVED** [review]
Congrats!! Nice work!

**Comment by Sof222** [comment]
Hi!

Thanks for approving this Pull Request. :)

Am I able to do another Pull Request, or should I wait for this one to be merged?

**Pull request merged**